_posts/2015-12-16-29.markdown

```diff
@@ -0,0 +1,283 @@
+
+There’s a lot of reasons for me to be in NYC, and a lot for Danger to stay in the UK. That coupled with [no simple][23] way for her to come over on my H1B means that we’re still not too sure about this. I’m currently on a H1B for the next 2 years, but also applying for an [O1 visa][24] which may mean if we marry she could come live in the US. Hard to see how this is going to turn out. Still a big question mark for us.
+
+* VISA renewed? O1? DM?
+---- 
+
+I spent all of last year listening to Rise Against, A Day to Remember and Billy Talent. I mellowed out this year, and seemed to listen to a lot of Fallout Boy and My Chemical Romance.
+
+I wonder if I’m less angry about absolute surveillance. I bought a series of prints to remind me that I should be very angry. I still only make playlists called “Sprint [number]” on Spotify, which I play on repeat all day. I’ve been doing that for years.
+
+---- 
+
+I’m 4 years into Artsy, and we’ve hit the [Dunbar number][25] this year.
+
+I feel like this year was when the Artsy mobile team really started to flourish. When it was created, dB and Laura joined from the web team to bolster numbers, and this year we filled it up enough to be able to stand on it’s own feet.
+
+To quote “[on being 28][26]”
+
+> Dropping the team down to one was a bit of a bummer. However, I got a promotion of sorts as I was the only one left.
+
+It’s like night and day now. I study management styles like I used to study programming languages. I’d recommend reading [Tribal Leadership][27], [The Year Without Pants][28] and [Leadership and self-deception][29]. Artsy is big enough to have politics, and I find it interesting to be a part of that.
+
+We spent the time on building team foundations like the [article in objc.io][30] helping us understand how we see ourselves. We’re a confident team, shipping code in the open.
+
+This time last year, we had only just started down the path to [Open By Default][31] with [Eidolon][32]. Now that’s [complete][33]. With help from the rest of the team we successfully opened the entire development process for every Artsy iOS app.
+
+Developing in The Open is the premise that hiding our code provides less value for us than keeping it open. It means showing process, design and source. It means making notes from meetings available, moving internal chats in the public and trying to make it accessible to as many as possible. Not everything is public, but the vast, vast majority is.
+
+It’s not a simple process, and it’s idealistic and utopian in its vision, but it can work for the majority of the things that someone would want me to work on. I hope it becomes the way everyone works in the future.
+
+We’ve shipped a lot of things I’m super proud of helping out in.
+
+ * Shipping an Apple TV app.
```

Comment:
There's a double indent of bullet points here. Intentional? :o (I've been reading the post through github markdown renderer: https://github.com/orta/orta.github.com/blob/29/_posts/2015-12-16-29.markdown)